app/services/notices/setup/submit-contact-type.service.js

```diff
@@ -103,6 +103,11 @@ async function _save(session, payload, yar) {
   session.contactName = payload.name
   session.contactType = payload.type
 
+  session.backLink = {
```

Comment:
this should be added in the initiate session service next to where we set the redirect url: 
https://github.com/DEFRA/water-abstraction-system/blob/main/app/services/notices/setup/initiate-session.service.js#L67